app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -188,6 +189,13 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
             super.onCreate(savedInstanceState)
         }
 
+        // Checks if in PiP mode, and if launched from external intents, to avoid unintended changes
+         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N && isInPictureInPictureMode && intent != null) {
+             // Exit PiP mode
+             moveTaskToBack(false)
+             startActivity(Intent(this, this::class.java).setFlags(FLAG_ACTIVITY_REORDER_TO_FRONT))
+         }
```

Comment:
` Unexpected indentation (9) (it should be 8) (cannot be auto-corrected)`

```suggestion
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N && isInPictureInPictureMode && intent != null) {
            // Exit PiP mode
            moveTaskToBack(false)
            startActivity(Intent(this, this::class.java).setFlags(FLAG_ACTIVITY_REORDER_TO_FRONT))
        }
```